lib/adal.js

```diff
@@ -662,7 +662,7 @@ var AuthenticationContext = (function () {
 
             if (requestInfo.requestType === this.REQUEST_TYPE.LOGIN) {
                 this._loginInProgress = false;
-                this._saveItem(this.CONSTANTS.STORAGE.LOGIN_ERROR, requestInfo.parameters.errorDescription);
+                this._saveItem(this.CONSTANTS.STORAGE.LOGIN_ERROR, requestInfo.parameters.error_description);
```

Comment:
Why did we add an underscore between "error" and "description"? It looks like the predominant style is to use camel casing.